jupyterhub_moss/templates/option_form.html

```diff
@@ -1,3 +1,23 @@
+{% macro resource_tab_footer(partitions, simple_only=false) %}
```

Comment:
```suggestion
{% macro resource_tab_footer(partitions, simple_only) %}
```

I think it is clearer to keep the arg explicit. It is not much more to write since the macro is only called twice